configure.ac

```diff
@@ -2605,6 +2605,13 @@ then
     AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_SHA3_SMALL"
 fi
 
+# SHAKE128
```

Comment:
General comment: SHAKE128 and SHAKE256 are missing from the benchmarks. Please add in a future PR.